tt_metal/impl/program/program_dispatch_utils.hpp

```diff
@@ -0,0 +1,112 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "tt_metal/impl/buffers/circular_buffer.hpp"
+#include "tt_metal/impl/device/device.hpp"
+#include "tt_metal/impl/kernels/kernel.hpp"
+#include "tt_metal/impl/program/program.hpp"
+#include "tt_metal/impl/dispatch/worker_config_buffer.hpp"
+
+namespace tt {
+
+namespace tt_metal {
+
+namespace program_utils {
+#define CQ_PREFETCH_CMD_BARE_MIN_SIZE tt::tt_metal::hal.get_alignment(tt::tt_metal::HalMemType::HOST)
```

Comment:
If we are momving/cleaning things up we should just get rid of this define I think. Not a blocker for this pr though.

Reply:
I've removed this define. We use the HAL directly instead.